xunit-autolabeler-v2/ast_parser/python/source_parsers/__init__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Pytest imports relative to the module root, while python itself
+# imports relative to the importing file's directory. To remedy this
+# inconsistency, we use project-root based imports (for greater clarity)
+# and add the project root to PYTHONPATH
+
+# import sys
```

Comment:
Suggestion:

Just remove them. I don't know what the `python_bootstrap.py` is. It's just confusing for other people.

Reply:
They're related to components that will soon (O(weeks)) be merged. If this is unnecessary for _those_ components, then I'll remove it outright.

Reply:
Can you remove them?

Reply:
+1 I'm pro deferring creating it until you need it. If you end up not using it you don't need to remember to clean it up :)

Reply:
Got it - removed.